openfl-tutorials/interactive_api/PyTorch_Kvasir_UNet/envoy/kvasir_shard_descriptor.py

```diff
@@ -61,42 +97,21 @@ def download_data(data_folder):
         os.system(f'unzip -n {zip_file_path.relative_to(Path.cwd())}'
                   f' -d {data_folder.relative_to(Path.cwd())}')
 
-    def __getitem__(self, index):
-        """Return a item by the index."""
-        name = self.images_names[index]
-        # Reading data
-        img = Image.open(self.images_path / name)
-        mask = Image.open(self.masks_path / name)
-        if self.enforce_image_hw is not None:
-            # If we need to resize data
-            # PIL accepts (w,h) tuple, not (h,w)
-            img = img.resize(self.enforce_image_hw[::-1])
-            mask = mask.resize(self.enforce_image_hw[::-1])
-        img = np.asarray(img)
-        mask = np.asarray(mask)
-        assert img.shape[2] == 3
-
-        return img, mask[:, :, 0].astype(np.uint8)
-
-    def __len__(self):
-        """Return the len of the dataset."""
-        return len(self.images_names)
-
     @property
     def sample_shape(self):
         """Return the sample shape info."""
-        return self._sample_shape
+        return ['300', '400', '3']
```

Comment:
Yeah, why we now hard code this? It is probably the only example with parametrized sample shape.
Kvasir is also probably the only example with a functioning dummy shard descriptor

Reply:
Make sense, was changed